### ISSUE TITLE
`@prefresh/vite@0.8.2`: require is not defined

### ISSUE THREAD

**Issue by pnfcre** [JoviDeCroock/prefresh#86]
Hi there,

After upgrading from `@prefresh/vite@0.8.1` to `@prefresh/vite@0.8.2`, the following lines are injected into the browser code, causing a reference error:

```js
require('@prefresh/vite/runtime');
const { compareSignatures } = require('@prefresh/vite/utils');
```

This can be reproduced by running the following:

```sh
$ yarn create vite-app test --template preact && cd test
$ yarn && yarn upgrade --latest && yarn dev
```

Everything works fine when pinning the version to `0.8.1`. Thanks! 👍

Kind regards,
Hampus Kraft.

**Comment by JoviDeCroock** [comment]
Hey,

Thanks for the issue report, I can see why this issue is popping up, the `prepublishOnly` step translates from `ESM` --> `CJS` but it seems to also translate the code within the string-literal which isn't needed. I'll see what I can do

**Comment by pnfcre** [comment]
Oh, I upgraded my Node version using `nvm` and opened a new shell without setting a new default! My bad, it works now. Thanks!

**Comment by pnfcre** [comment]
Hi there, it's me again, sorry. Looks like it temporarily worked again for some reason, but I now can reproduce the same error again, despite using Node `14.4.0` 🤔

**Comment by JoviDeCroock** [comment]
Hey @pnfcre I published `@prefresh/vite` v0.8.3 which should fix this issue! Thanks for notifying me about this and we'll try to upstream it to `bundt`/`rewrite-imports` asap
